web-ng/tslint.json

```diff
@@ -1,4 +1,8 @@
 {
+  "rules": {
+    "no-any": false,
```

Comment:
Let's define this on a per-line basis if possible. Existing warnings are fixed in #203.

Reply:
Done